php/hireinsocial/db/migrations/2019/02/Version20190203180027.php

```diff
@@ -8,23 +8,26 @@
 /**
  * Auto-generated Migration: Please modify to your needs!
  */
-final class Version20190107165045 extends AbstractMigration
+final class Version20190203180027 extends AbstractMigration
```

Comment:
I should not change existing migrations but it's very early stage, this migration wasn't used anywhere.